cli/src/pcluster/schemas/imagebuilder_schema.py

```diff
@@ -131,34 +136,11 @@ def validate_security_group_ids(self, value):
 # ---------------------- Dev Settings Schema ---------------------- #
 
 
-class ChefCookbookSchema(BaseSchema):
-    """Represent the schema of the ImageBuilder chef cookbook."""
-
-    url = fields.Str()
-    json = fields.Str()
-
-    @post_load()
-    def make_resource(self, data, **kwargs):
-        """Generate resource."""
-        return ChefCookbook(**data)
-
-    @validates("json")
-    def validate_json(self, value):
-        """Validate json."""
-        if value and not validate_json_format(value):
-            raise ValidationError(
-                message="The Json in ChefCookbook '{0}' is invalid, check the json format.".format(value)
-            )
-
-
-class DevSettingsSchema(BaseSchema):
+class DevSettingsSchema(BaseDevSettingsSchema):
```

Comment:
let's avoid the same class name, let's call them ImagebuilderDevSettingsSchema and ClusterDevSettingsSchema

Reply:
Done